cluster/src/dunit/scala/io/snappydata/metrics/SnappyMetricsSystemDUnitTest.scala

```diff
@@ -60,16 +53,15 @@ class SnappyMetricsSystemDUnitTest(s: String)
         s" -peer-discovery-port=$port -client-port=$netPort")
     pw.close()
     val pw1 = new PrintWriter(new File(s"$confDir/leads"))
-    pw1.write(s"localhost -locators=localhost[$port] -dir=$snappyProductDir/work/lead1\n")
-    pw1.write(s"localhost -locators=localhost[$port] -dir=$snappyProductDir/work/lead2")
+    pw1.write(s"localhost -locators=localhost[$port] " +
+        s"-dir=$snappyProductDir/work/lead1 -spark.ui.port=3333\n")
+    pw1.write(s"localhost -locators=localhost[$port] " +
+        s"-dir=$snappyProductDir/work/lead2 -spark.ui.port=3334")
     pw1.close()
     val pw2 = new PrintWriter(new File(s"$confDir/servers"))
-    pw2.write(s"localhost -locators=localhost[$port] -client-port=$netPort2" +
-        s" -dir=$snappyProductDir/work/server1\n")
-    pw2.write(s"localhost -locators=localhost[$port] -client-port=$netPort3" +
-        s" -dir=$snappyProductDir/work/server2\n")
-    pw2.write(s"localhost -locators=localhost[$port] -client-port=$netPort4" +
-        s" -dir=$snappyProductDir/work/server3")
+    pw2.write(s"localhost -locators=localhost[$port] -dir=$snappyProductDir/work/server1\n")
```

Comment:
Don't need to specify `-dir` property. Scripts automatically create these directories.

Reply:
I need that option for getting full and short directory names for members work directory.